File: VentasVB/frmVentas.vb
Imports Npgsql
Public Class frmVentas
	Private Sub frmVentas_Load(sender As Object, e As EventArgs) Handles MyBase.Load
		Dim con As NpgsqlConnection = New NpgsqlConnection("Server=127.0.0.1;Port=5432;Database=Ventas;User Id=postgres;Password=jordi123;")
		CuPuntoVenta1.BringToFront()

	End Sub

	Private Sub proveedorToolStripMenuItem_Click(sender As Object, e As EventArgs) Handles proveedorToolStripMenuItem.Click
		CuProveedor1.BringToFront()

	End Sub

	Private Sub sucursalToolStripMenuItem_Click(sender As Object, e As EventArgs) Handles sucursalToolStripMenuItem.Click
		CuCrudSucursal1.BringToFront()

	End Sub

	Private Sub cargoToolStripMenuItem_Click(sender As Object, e As EventArgs) Handles cargoToolStripMenuItem.Click
		CuCrudCargos1.BringToFront()

	End Sub

	Private Sub turnoToolStripMenuItem_Click(sender As Object, e As EventArgs) Handles turnoToolStripMenuItem.Click
		CuTurno1.BringToFront()
	End Sub

	Private Sub empleadoToolStripMenuItem_Click(sender As Object, e As EventArgs) Handles empleadoToolStripMenuItem.Click
		CuEmpleado1.BringToFront()

	End Sub

	Private Sub categoriaToolStripMenuItem_Click(sender As Object, e As EventArgs) Handles categoriaToolStripMenuItem.Click
		CuCategoria1.BringToFront()

	End Sub

	Private Sub clienteToolStripMenuItem_Click(sender As Object, e As EventArgs) Handles clienteToolStripMenuItem.Click
		CuCliente1.BringToFront()

	End Sub

	Private Sub productoToolStripMenuItem_Click(sender As Object, e As EventArgs) Handles productoToolStripMenuItem.Click
		CuProducto1.BringToFront()

	End Sub

	Private Sub sucursalProveedorToolStripMenuItem_Click(sender As Object, e As EventArgs) Handles sucursalProveedorToolStripMenuItem.Click
		CuProveedorSucursal1.BringToFront()

	End Sub

	Private Sub nuevaVentaToolStripMenuItem_Click(sender As Object, e As EventArgs) Handles nuevaVentaToolStripMenuItem.Click
		CuPuntoVenta1.BringToFront()
	End Sub

	Private Sub listaVentasToolStripMenuItem_Click(sender As Object, e As EventArgs) Handles listaVentasToolStripMenuItem.Click
		CuVentas1.BringToFront()

	End Sub

	Private Sub frmVentas_FormClosed(sender As Object, e As FormClosedEventArgs) Handles MyBase.FormClosed
		Application.Exit()

	End Sub
End Class

File: Ventas/frmVentas.cs
using System;
using System.Windows.Forms;

namespace Ventas
{
    public partial class frmVenta : Form
    {
        public frmVenta()
        {
            InitializeComponent();
            cuPuntoVenta1.BringToFront();
        }

        private void proveedorToolStripMenuItem_Click(object sender, EventArgs e)
        {
            cuProveedor1.BringToFront();
        }

        private void sucursalToolStripMenuItem_Click(object sender, EventArgs e)
        {
            cuCrudSucursal1.BringToFront();
        }

        private void cargoToolStripMenuItem_Click(object sender, EventArgs e)
        {
            cuCrudCargos1.BringToFront();
        }

        private void turnoToolStripMenuItem_Click(object sender, EventArgs e)
        {
            cuTurno1.BringToFront();
        }

        private void empleadoToolStripMenuItem_Click(object sender, EventArgs e)
        {
            cuEmpleado1.BringToFront();
        }

        private void categoriaToolStripMenuItem_Click(object sender, EventArgs e)
        {
            cuCategoria1.BringToFront();
        }

        private void clienteToolStripMenuItem_Click(object sender, EventArgs e)
        {
            cuCliente1.BringToFront();
        }

        private void productoToolStripMenuItem_Click(object sender, EventArgs e)
        {
            cuProducto1.BringToFront();
        }

        private void sucursalProveedorToolStripMenuItem_Click(object sender, EventArgs e)
        {
            cuProveedorSucursal1.BringToFront();
        }

        private void nuevaVentaToolStripMenuItem_Click(object sender, EventArgs e)
        {
            cuPuntoVenta1.BringToFront();
        }

        private void listaVentasToolStripMenuItem_Click(object sender, EventArgs e)
        {
            cuVentas1.BringToFront();
        }

        private void frmVenta_FormClosed(object sender, FormClosedEventArgs e)
        {
            Application.Exit();
        }
    }
}
